package.json

```diff
@@ -8,5 +8,8 @@
   },
   "scripts": {
     "test": "echo \"wikit tests\" && exit 0"
```

Comment:
In order for our CI to be able to pick up the sub packages we either have to tell it to `lerna run test` or instruct the root level package's test script (it is currently hooked up to CI) to do it instead.

I'm in favor of the latter (even if this is not what the lerna authors intended) because everyone knows how to interpret a `package.json` but only few will expect a test target other than `npm test` from being run in CI.

Reply:
Yup, I agree. Changed the test script to invoke lerna.